neuro_evolution_ctrnn/train.py

```diff
@@ -34,5 +36,5 @@ def parse_args(args=None):
     initialise everything that is not pickle-able before this point. Especially the DAEP-toolbox is not 
     pickle-able. 
     """
-    os.makedirs(args.result_path)
+    os.mkdir(args.result_path)
```

Comment:
That created an error for me because the directory `CTRNN_Simulation_Results` was not yet created and therefore the subdirectory containing the new training folder cannot be created. Going back to `os.makedirs(args.result_path)` fixes this

Reply:
makedirs soll einen fehler werfen, wenn das directory nicht da ist, um zu vermeiden, dass man versehentlich etwas falsches angegeben hat. 

in diesem Falle hätte "CTRNN_Simulation_Results" durch ein git-clone angelegt worden sein, damit du die ergebnisse am ende commiten kannst. Wenn CTRNN_Simulation_Results automatisch durch das script angelegt wird, dann würdest du erst merken dass etwas nicht stimmt, wenn das script fertig ist.

Reply:
Resolved :)